back/.env

```diff
@@ -1,2 +1,7 @@
 NODE_ENV=development
 PORT=8081
+SOCKET_PORT=3000
```

Comment:
We will use same express server to create socket channel. Check this [PR on planning-poker](https://github.com/Lemoncode/planning-poker-example/pull/84/files) when we remove SOCKET_PORT. Move the create socket server functionality to ./src/core/servers/socket.server.ts